Source/Core/Core/NetPlayServer.cpp

```diff
@@ -125,11 +128,16 @@ void NetPlayServer::ThreadFunc()
 
 		ENetEvent netEvent;
 		int net;
+		if (m_traversal_client)
+			m_traversal_client->HandleResends();
+		net = enet_host_service(m_server, &netEvent, 1000);
+		while (!m_async_queue.Empty())
 		{
-			std::lock_guard<std::recursive_mutex> lks(m_crit.send);
-			if (m_traversal_client)
-				m_traversal_client->HandleResends();
-			net = enet_host_service(m_server, &netEvent, 4);
```

Comment:
On the client side, there is still the 4ms timeout. Maybe we should also increase this.